.env.sample

```diff
@@ -10,4 +10,7 @@ ERC4337_TEST_NODE_URL=
 ERC4337_TEST_SINGLETON_ADDRESS=
 ERC4337_TEST_SAFE_FACTORY_ADDRESS=
 # (Optional) Tells the test runner which Safe Singleton Contract to use for testing: Safe or SafeL2. Defaults to Safe.
-SAFE_CONTRACT_UNDER_TEST="Safe"
\ No newline at end of file
+SAFE_CONTRACT_UNDER_TEST="Safe"
+# Used for compiling with different solidity testing
+SOLIDITY_VERSION= #
+SOLIDITY_SETTINGS= # Example: '{"viaIR":false,"optimizer":{"enabled":true, "details": {"yul": true}}}'
```

Comment:
`"details": {"yul": true}` seems to be only required in solidity <0.6.0:

>          // The new Yul optimizer. Mostly operates on the code of ABI coder v2
>         // and inline assembly.
>         // It is activated together with the global optimizer setting
>         // and can be deactivated here.
>         // Before Solidity 0.6.0 it had to be activated through this switch.
>         "yul": false,

        
 from https://arc.net/l/quote/qslzbszq

Reply:
This was just an example, but it makes sense to update it. Done 👍🏾

Reply:
It comes back, as the `optimizerSteps` in `yulDetails` are only accepted if `yul` is true. Else:

> Providing yulDetails requires Yul optimizer to be enabled.

Required for `coverage` while using `v0.8.19`.